### ISSUE TITLE
** 8.21 - issue with CoCCode **

### ISSUE THREAD

**Issue by michelleogburn** [abtassociates/LSASampleCode#405]
### Issue in the spec
![image-20200804-145230](https://user-images.githubusercontent.com/20728145/89370193-ff99e100-d694-11ea-87a4-31d8d65809af.png)

### Issue in the sample code
![image-20200804-145452](https://user-images.githubusercontent.com/20728145/89370220-10e2ed80-d695-11ea-9779-38ad57420048.png)

We do need to use `coc.CoCCode = rpt.ReportCoC` as a restriction.

**Comment by MollyMcEvilley** [comment]
Updated the specs so that they say 'and CoCCode is not NULL.' 

It isn't necessary that the CoCCode match ReportCoC -- as long as there is a CoC code (they might have been served in another CoC and that's fine), we're not counting it as a problem.  

There WAS an issue in the code related to EnrollmentCoC -- ANY null record could have triggered an issue and we only want to count enrollments where there is no CoCCode associated with the enrollment at all.  I have updated that.
